samples/session-replay-mobile-schema/session-replay/shape.json

```diff
@@ -0,0 +1,17 @@
+{
+    "id": 1,
+    "x": 10,
+    "y": 10,
+    "width": 20,
+    "height": 30,
+    "opacity": 1.0,
+    "cornerRadius": 0.0,
+    "shapeStyle": {
```

Comment:
Consider naming `backgroundColor` to match the browser for HTML (not XML).

Reply:
same as above, @ncreated need your opinion here also.

Reply:
Thing is, `shapeStyle` bundles more than background color: stroke width, stroke color, fill color, corner radius, opacity, ... . It could make sense to call `fillColor` a `backgroundColor`, but IMO not the `shapeStyle`.

Reply:
Similarly there are a few more styles I was thinking of, like fill and stroke being SVG based as well.
Anyway no big deal at this point.